lib/BuildSystem/BuildSystem.cpp

```diff
@@ -1009,7 +1013,11 @@ class FilteredDirectoryContentsTask : public Task {
       std::string filename = llvm::sys::path::filename(it->path());
       bool excluded = false;
       for (auto pattern : filterStrings) {
+#if defined(_WIN32)
+        if (PathMatchSpecA(filename.c_str(), pattern.data())) {
```

Comment:
Might be worth putting this into a method in PlatformUtility.   Would be nice to try to keep the _WIN32 ifdefs contained to that layer as much as possible.